p2p_wallet/Scenes/CreateOrRestoreWallet/Old/CreateOrRestoreWalletHandler.swift

```diff
@@ -9,6 +9,11 @@ import Foundation
 import SolanaSwift
 
 protocol CreateOrRestoreWalletHandler {
```

Comment:
I don't like `CreateOrRestoreWalletHandler`. It would better to move all logic to `WalletManager` class. `AppCoordinator` will listen this class and navigate users to required screens